<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>画圆Demo</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSlider" name="radiusSetter">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>20</y>
      <width>22</width>
      <height>120</height>
     </rect>
    </property>
    <property name="minimum">
     <number>30</number>
    </property>
    <property name="maximum">
     <number>60</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="radius">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>30</y>
      <width>75</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>绘图半径</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setRadius">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>100</y>
      <width>80</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>设置</string>
    </property>
   </widget>
   <widget class="QSlider" name="circleRadiusSetter">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>150</y>
      <width>22</width>
      <height>120</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>35</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="circleRadius">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>160</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>圆半径</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addCircle">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>240</y>
      <width>80</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>添加圆</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cmpChecker">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>280</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>真实值对比</string>
    </property>
   </widget>
   <widget class="QPushButton" name="performanceTest">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>300</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>算法性能测试</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="original">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>330</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>中点算法</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="integer">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>360</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>极坐标算法</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="optimized">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>390</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>中点优化算法</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>659</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
